components/array-object-table/src/index.js

```diff
@@ -53,11 +53,36 @@ import { rowsFromArray, titlesFromFields } from '@govuk-frederic/utils';
  * 
  * <ArrayObjectTable fields={fields} array={array} title={title} skipEmptyRows hideWithNoValues/>;
  * ```
+ * 
+ * With object transform and default transform
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two', transform: value => value ? value.toLowerCase() : '' },
+ *   { key: 'three', heading: 'Three' },
+ *   { key: 'three', heading: 'Four', transform: value => value ? value.toLowerCase() : '*' },
+ * ];
+ * const array = [
+ *   {one: 'One', two: 'Two'},
+ * ];
+ * const title = ['Heading'];
+ * const defaultTransform = value => (value || '-');
+ *
+ * <ArrayObjectTable fields={fields} array={array} title={title} skipEmptyRows hideWithNoValues defaultTransform={defaultTransform}/>
+ * ```
  * */
-const ArrayObjectTable = ({ fields, array, hideWithNoValues, skipEmptyRows, title, ...props }) => {
-  let rows = rowsFromArray(array, fields, skipEmptyRows);
+const ArrayObjectTable = ({
+  fields,
+  array,
+  hideWithNoValues,
+  skipEmptyRows,
+  title,
+  defaultTransform,
+  ...props
+}) => {
+  let rows = rowsFromArray(array, fields, skipEmptyRows, defaultTransform);
   if (!rows.length && !hideWithNoValues) {
-    rows = rowsFromArray([{}], fields, false);
+    rows = rowsFromArray([{}], fields, false, defaultTransform);
```

Comment:
Instead of passing `false` here, it would be good to name the variable and give it a value so someone knows what they are looking at.

Reply:
perhaps : ) altho, I just noticed it's there on the line above to... so...